jenkins-slaves/ansible/roles/kie/tasks/main.yml

```diff
@@ -153,49 +153,6 @@
     path: /home/jenkins/git-repos
```

Comment:
Do we need still the directory to be created?

Reply:
Oh. It slipped through. I removed the Jenkins permissions for git-repos, but missed that one. Correcting right now.